src/app/settings/admin/opa/template.html

```diff
@@ -19,4 +19,9 @@
   <km-tab label="Default Constraints">
     <km-default-constraint-list></km-default-constraint-list>
   </km-tab>
+  <km-tab label="Allowed Registries"
+          *ngIf="isEE()">

```

Comment:
The idea of using dynamic ce/ee modules was to not use this kind of logic and let the router load proper ce/ee version instead. It should be hidden inside and the only thing here should be the `router-outlet`.

Reply:
Yes, agree, that was my first thought too, but I did not manage to load this tab label from the router-outlet for some reason.

Reply:
I think there might be a problem with `km-tab-card` not recognizing the `km-tab` if it comes from the router outlet. I would check that first.

Reply:
Something like this, yeah. The tab isn't in the `inputTabs` list in `km-tab-card`, even though it seems that `km-tab` is "receiving" it. Therefor it isn't rendered.
Investigated a bit more and tried out a lot, but still didn't manage to create a working solution. 🤷 

Is someone having an idea and wants to take a look on this problem?

Reply:
I'll take a look

Reply:
Added dynamic tab support. Should be ok now.

Reply:
Thanks, looks good. 👍 
I spot a lot of parts, that I've also tried, but obviously not in the right combination.